activerecord/lib/active_record/enum.rb

```diff
@@ -218,6 +218,13 @@ def assert_valid_enum_definition_values(values)
           MSG
           raise ArgumentError, error_message
         end
+
+        if values.is_a?(Hash) && values.keys.any?(&:blank?) || values.is_a?(Array) && values.any?(&:blank?)
+          error_message = <<~MSG
+            Enum label name must not be blank.
+          MSG
+          raise ArgumentError, error_message
```

Comment:
Here I think we can simplify its syntax by not defining `error_message` and simply returning:
```suggestion
          raise ArgumentError, "Enum label name must not be blank."
```

Reply:
Indeed, it's a short string 👍